db/migrate/20150122082200_add_scopes_to_oauth_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopesToOauthApplications < ActiveRecord::Migration
```

Comment:
@danielroseman - you're right. i've removed that migration. in the CHANGELOG.md, they've called it a backwards incompatible change, so i was lead to believe that it is needed unconditionally.